test/expected/partitionwise-13.out

```diff
@@ -0,0 +1,884 @@
+         Output: _hyper_3_7_chunk."time", _hyper_3_7_chunk.device, (avg(_hyper_3_7_chunk.temp))
+         Sort Key: _hyper_3_7_chunk."time", _hyper_3_7_chunk.device
+         ->  Append
+               ->  HashAggregate
+                     Output: _hyper_3_7_chunk."time", _hyper_3_7_chunk.device, avg(_hyper_3_7_chunk.temp)
+                     Group Key: _hyper_3_7_chunk."time", _hyper_3_7_chunk.device
+                     ->  Seq Scan on _timescaledb_internal._hyper_3_7_chunk
+                           Output: _hyper_3_7_chunk."time", _hyper_3_7_chunk.device, _hyper_3_7_chunk.temp
+               ->  HashAggregate
+                     Output: _hyper_3_8_chunk."time", _hyper_3_8_chunk.device, avg(_hyper_3_8_chunk.temp)
+                     Group Key: _hyper_3_8_chunk."time", _hyper_3_8_chunk.device
+                     ->  Seq Scan on _timescaledb_internal._hyper_3_8_chunk
+                           Output: _hyper_3_8_chunk."time", _hyper_3_8_chunk.device, _hyper_3_8_chunk.temp
+(16 rows)
+
+-- Applying the time partitioning function should also allow push-down
+-- on open dimensions
+:PREFIX
+SELECT time_func(time), device, avg(temp)
+FROM hyper_timepart
+GROUP BY 1, 2
+ORDER BY 1, 2
+LIMIT 10;
+                                                              QUERY PLAN                                                              
+--------------------------------------------------------------------------------------------------------------------------------------
+ Limit
+   Output: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device, (avg(_hyper_3_7_chunk.temp))
+   ->  Merge Append
+         Sort Key: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device
+         ->  GroupAggregate
+               Output: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device, avg(_hyper_3_7_chunk.temp)
+               Group Key: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device
+               ->  Incremental Sort
+                     Output: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device, _hyper_3_7_chunk.temp
+                     Sort Key: (time_func(_hyper_3_7_chunk."time")), _hyper_3_7_chunk.device
+                     Presorted Key: (time_func(_hyper_3_7_chunk."time"))
+                     ->  Index Scan Backward using _hyper_3_7_chunk_hyper_timepart_expr_idx on _timescaledb_internal._hyper_3_7_chunk
+                           Output: time_func(_hyper_3_7_chunk."time"), _hyper_3_7_chunk.device, _hyper_3_7_chunk.temp
+         ->  GroupAggregate
+               Output: (time_func(_hyper_3_8_chunk."time")), _hyper_3_8_chunk.device, avg(_hyper_3_8_chunk.temp)
+               Group Key: (time_func(_hyper_3_8_chunk."time")), _hyper_3_8_chunk.device
+               ->  Incremental Sort
+                     Output: (time_func(_hyper_3_8_chunk."time")), _hyper_3_8_chunk.device, _hyper_3_8_chunk.temp
+                     Sort Key: (time_func(_hyper_3_8_chunk."time")), _hyper_3_8_chunk.device
```

Comment:
This is different from PG12. Have you checked it?

Reply:
yes